jest.setup-after-env.ts

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import '@testing-library/jest-native/extend-expect';
```

Comment:
Using this file instead of just placing `'@testing-library/jest-native/extend-expect'` directly in Jest `setupFilesAfterEnv` like before, allows Typescript type checking of the added `expect` matchers.